c/python/namedtuple.cc

```diff
@@ -132,4 +132,14 @@ onamedtuple::onamedtuple(const onamedtupletype& type) {
 }
 
 
+void onamedtuple::copy_v() {
+  PyObject* v1 = PyTuple_GetSlice(v, 0, PyTuple_Size(v));
+  PyTypeObject* type = Py_TYPE(v);
+  Py_TYPE(v1) = type;
+  Py_INCREF(type);
+  Py_DECREF(v);
+  v = v1;
```

Comment:
`Py_SETREF` macro is a safer alternative

Reply:
Nice, can't see this macro anywhere in the docs, and it seems that we're also not using it in dt? The only place where there is some info is here https://github.com/python/cpython/blob/master/Include/cpython/object.h#L353-L370 Looks like a much better approach.

Reply:
Yeah, I can't find it in the docs either... Weird.